src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -1063,14 +1074,6 @@ private static unsafe void EnsureInitialized()
                         throw new Win32Exception();
                     }
 
-                    s_euid = Interop.Sys.GetEUid();
-                    s_egid = Interop.Sys.GetEGid();
-                    s_groups = Interop.Sys.GetGroups();
```

Comment:
I really like this change as it removes 3 sys-calls from the initialization logic that is used for every process, no matter if it's needed or not. 👍